Importer/EventListener/InterpreterExceptionSubscriber.php

```diff
@@ -50,6 +51,8 @@ private function resolveInterpreterExceptionMessage(InterpreterException $except
             return "Row is expected to contain {$exception->getExpectedSize()}, found to have {$exception->getFoundSize()} fields instead.";
         } elseif ($exception instanceof InvalidRowException) {
             return $this->buildInvalidRowExceptionMessage($exception);
+        } elseif ($exception instanceof BooleanFormatException) {
```

Comment:
We should not be showing exception messages to the user either way. If we had a generic message, we could build it with parameters passed from a geneic exception. This could even allow translation etc.